feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="overview-price">
+  <ng-container [ngSwitch]="getOverviewPriceType()">
+    <ng-container *ngSwitchCase="overviewPriceType.PRICE_ONLY"
+      ><span>{{ calculateTotal() }}</span></ng-container
```

Comment:
Currency support will be added, right?
(Not part of this ticket I guess)

Reply:
Not a part of this ticket. It clearly states, that here we only should add the values, without any styling whatsoever.